README.md

```diff
@@ -1,4 +1,29 @@
-# Gateway API Inference Extension 
+# Gateway API Inference Extension
+
+This project offers tools for [AI Inference], enabling developers to build [AI Gateways and Inference Frameworks].
+
+[AI Inference]:https://research.ibm.com/blog/AI-inference-explained
+[AI Gateways and Inference Frameworks]:#concepts-and-definitions
+
+## Concepts and Definitions
+
+AI/ML is evolving rapidly, and [Inference] goes beyond basic networking to include complex traffic routing and optimizations. Below are key terms for developers:
+
+- **GIE**: Short for "Gateway API Inference Extension".
+- **Scheduler**: Manages routing in inference, basing decisions on routing rules (e.g., "model name") and `Metrics and Capabilities` from [Model Serving Platforms].
```

Comment:
The scheduler currently doesn't make "request routing" decisions based on the model name, the current implementation assumes that the pool supports a single base model, and all servers within the pool can serve any of the defined adapters on the pool.

Reply:
:+1: 

```suggestion
- **Scheduler**: Manages routing in inference, basing decisions on routing rules and `Metrics and Capabilities` from [Model Serving Platforms].
```